docs/source/notebooks/pcca.ipynb

```diff
@@ -66,7 +100,16 @@
    "outputs": [],
    "source": [
     "msm = sktime.markov.msm.MaximumLikelihoodMSM() \\\n",
-    "    .fit(assignments, lagtime=1).fetch_model()"
+    "    .fit(assignments, lagtime=1).fetch_model()\n",
+    "\n",
+    "print(f\"Number of states: {msm.n_states}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plotting the transition matrix, it is not immediately obvious that there are two metastable sets. With a connectivity threshold discarding weaker connections, one can guess it (i.e., the degree of connectedness within one metastable set is much higher than between them), but this is not always possible and also not very rigorous."
```

Comment:
you could visualize the transmat values and look for block diagonal structure, right?

Reply:
highly depends on the initialization strategy of the clustering. it only works if you assume that close indices == close centers spatially

Reply:
true